download_script.py

```diff
@@ -0,0 +1,50 @@
+import requests
+import argparse
+
+# help function for better understanding of the scripts functionality
+
+description = """  This script is used to download all the pickle/dataset files needed to train, evaluate and predict using the model.
```

Comment:
That seems to be a typo as both tokenizer and dataset files are pickles:
```suggestion
description = """  This script is used to download all the tokenizer/dataset pickle files needed to train, evaluate and predict using the model.
```

By the way, this script should rather be named `get_pickles.py` which would be more self-explanatory.